openapi/components/schemas/RiskScoreAllowListRecord.yaml

```
@@ -0,0 +1,33 @@
+type: object
+description: Risk score allow list record.
+required:
+  - type
+  - value
+properties:
+  id:
+    type: string
+    description: ID of the allow list record.
+    readOnly: true
+    maxLength: 50
+    example: rsal_0YVF9605RKC62BP14NE2R7V2XT
+  type:
+    type: string
+    description: Type of exclusion from adding to blocklist.
```

suggestion: it may be good to add more detail here
```suggestion
    description: |-
      Type of data to exclude from risk score checks.
      Data types that are added to this field are excluded from risk score checks and are not added to blocklists.
```